test/unit-test/core_mqtt_utest.c

```diff
@@ -1974,6 +2058,7 @@ void test_MQTT_ProcessLoop_handleKeepAlive_Error_Paths( void )
     MQTTFixedBuffer_t networkBuffer;
     ProcessLoopReturns_t expectParams;
 
+    setupNetworkBuffer( &networkBuffer );
```

Comment:
Interesting that this call wasn't there before, but it's test was probably passing. Presumably depended on prior calls earlier in the suite. Thanks for the addition.